src/vivarium_public_health/treatment/intervention.py

```diff
@@ -6,21 +6,45 @@
 
 
 class Intervention(Exposure):
-    """A component for modeling access to an intervention. The modeling and implementation of
-    this component is similar to that of a risk factor where the risk is a loack or not having
-    access to the intervention. Interventions should be configured with in the format of
+    """A model for an intervention defined by coverage (access to intervention).
+
+    This component models access to an intervention as a dichotomous exposure where
+    simulants are either covered or uncovered by the intervention. The modeling
+    and implementation of this component is similar to that of a risk factor where
+    the risk is a lack of or not having access to the intervention.
```

Comment:
I think talking about modeling it like a lack of a risk here makes it more confusing. I suggest removing this sentence.